.github/actions/templates/publishModule/scripts/Publish-ModuleToTemplateSpec.ps1

```diff
@@ -135,11 +135,11 @@ function Publish-ModuleToTemplateSpec {
         if ($PSCmdlet.ShouldProcess("Template spec [$componentTemplateSpecName] version [$newVersion]", 'Publish')) {
             $templateSpecInputObject = @{
                 ResourceGroupName = $componentTemplateSpecRGName
-                Name              = $componentTemplateSpecName
+                Name              = $componentTempRgteSpecName
                 Version           = $newVersion
                 Description       = $componentTemplateSpecDescription
                 Location          = $componentTemplateSpecRGLocation
-                TemplateFile      = $templateFilePath
+                TemplateFile      = $templateFilePRgh
```

Comment:
Not sure what this is. Seems a search and replace gone wrong?

Reply:
Have you had 'insert' enabled?

Reply:
seems to be fixed